src/components/reusable/timepicker/timepicker.scss

```diff
@@ -0,0 +1,160 @@
+
+:host {
+  display: inline-block;
+}
+
+.time-picker {
+  width: 147px;
+}
+
+.label-text {
+  @include typography.type-ui-03;
+  text-transform: uppercase;
+  font-weight: 500;
+  display: block;
+  padding: 0;
+  margin-bottom: 8px;
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+  }
+}
+
+.required {
+  color: var(--kd-color-text-error);
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+  }
+}
+
+.input-wrapper {
+  position: relative;
+}
+
+slot[name='icon']::slotted(*) {
+  display: block;
+}
+
+input {
+  @include typography.type-body-02;
+  -webkit-appearance: none;
+  appearance: none;
+  border: 1px solid var(--kd-color-border-ui);
+  border-radius: 4px;
+  height: 48px;
+  padding: 10px 16px 10px 16px;
+  width: 167px;
+  color: var(--kd-color-text-primary);
+  transition: background-color 150ms ease-out, border-color 150ms ease-out,
+    outline-color 150ms ease-out;
+  background: var(--kd-color-background-inverse);
+  outline: 2px solid transparent;
+  outline-offset: 2px;
+
+  &.size--sm {
+    height: 32px;
+    padding: 2px 16px;
+  }
+
+  &.size--lg {
+    height: 56px;
+    padding: 14px 16px;
+  }
+
+  &[invalid] {
+    border-color: var(--kd-color-border-error);
+  }
+
+  &[disabled] {
+    color: var(--kd-color-text-disabled);
+    border-color: var(--kd-color-border-ui-disabled);
+  }
+
+  &:hover {
+    border-color: var(--kd-color-border-ui-hover);
+  }
+
+  &:focus {
+    outline-color: var(--kd-color-border-focus);
+  }
+
+  &.icon--left {
+    padding-left: 48px;
+  }
+}
+
+.error {
+  @include typography.type-ui-03;
+  display: flex;
+  align-items: center;
+  color: var(--kd-color-text-error);
+  margin-top: 8px;
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+  }
+}
+
+.error-icon {
+  position: absolute;
+  top: 50%;
+  right: -1px;
+  left: auto;
+  transform: translateY(-50%);
+  color: var(--kd-color-text-error);
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+  }
+}
+
+.warn {
+  @include typography.type-ui-03;
+  display: flex;
+  align-items: center;
+  color: var(--kd-color-text-secondary);
+  margin-top: 8px;
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+  }
+}
+
+.clear {
+  border: none;
+  background: none;
+  padding: 0;
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  right: -1px;
+  transform: translateY(-50%);
+  cursor: pointer;
+  color: var(--kd-color-text-primary);
+  background: var(--kd-color-background-inverse);
+  display: block;
+
+  kd-icon {
+    display: block;
+  }
+}
+
+.caption {
+  @include typography.type-ui-03;
+  margin-top: 8px;
+  color: var(--kd-color-text-secondary);
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+    border-color: var(--kd-color-border-ui-disabled);
+  }
+}
+// Change clock icon (22*22 fixed) according to UX: It won't appear at firefox, safari natively.
+input[type='time']::-webkit-calendar-picker-indicator {
```

Comment:
This doesn't show in Firefox either. I'd suggest just leaving it out since we agreed to go native.

Reply:
ok so no need to match with UX clock icon right ? because on UX its shows different icon than native one so just applied same

Reply:
Removed this and keep the native one only.